packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -461,8 +461,8 @@ export class AotPlugin implements Tapable {
             if (this.skipCodeGeneration) {
               this._lazyRoutes[k] = lazyRoute;
             } else {
-              const lr = path.relative(this.basePath, lazyRoute.replace(/\.ts$/, '.ngfactory.ts'));
-              this._lazyRoutes[k + '.ngfactory'] = path.join(this.genDir, lr);
+              const lr = path.relative(this.basePath, lazyRoute.replace(/(\.d)?\.ts$/, '.ngfactory.ts'));
+              this._lazyRoutes[k + '.ngfactory'] = path.join(this.genDir, path.join('/', lr));
```

Comment:
I understand the change on the line before (which LGTM), but I'm curious for why do you join `/` here and what does that mean for Windows users?